tests/e2e/test_linker_run.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import pytest
+
+from linker.utilities.data_utils import load_yaml
+from linker.utilities.slurm_utils import is_on_slurm
+
+SPECIFICATIONS_DIR = Path("tests/e2e/specifications")
+
+
+@pytest.mark.slow
+@pytest.mark.skipif(
+    not is_on_slurm(),
+    reason="Must be on slurm to run this test.",
+)
+@pytest.mark.parametrize(
+    "pipeline_specification, input_data, computing_environment",
+    [
+        # local
+        (
+            "pipeline.yaml",
+            "input_data.yaml",
+            "environment_local.yaml",
+        ),
+        # slurm
+        (
+            "pipeline.yaml",
+            "input_data.yaml",
+            "environment_slurm.yaml",
+        ),
+    ],
+)
+def test_linker_run(pipeline_specification, input_data, computing_environment, capsys):
+    """e2e tests for 'linker run' command"""
+    # Create a temporary directory to store results. We cannot use pytest's tmp_path fixture
+    # because other nodes do not have access to it.
+    with tempfile.TemporaryDirectory(dir="tests/e2e/") as results_dir:
```

Comment:
I can't use the `test_dir` fixture or the normal tmpdir b/c other slurm nodes need access to it (for the slurm pipelines).